client/src/helpers.js

```diff
@@ -125,6 +125,22 @@ class Helpers {
   hasIntent(path) {
     return !!path && path.toLowerCase().includes('#?intent=');
   }
+
+  deSanitizeParamsMap(paramsMap) {
+    return Object.entries(paramsMap).reduce((sanitizedMap, paramPair) => {
+      sanitizedMap[this.deSanitizeParam(paramPair[0])] = this.deSanitizeParam(paramPair[1]);
+      return sanitizedMap;
+    }, {});
+  }
+
+  deSanitizeParam(param = '') {
```

Comment:
I think we could extend this list with an ampersand ('&') and backtick ('`') https://stackoverflow.com/a/48226843

Reply:
We are de-sanitising(reverting) the sanitisation that we do in another place [here](https://github.com/SAP/luigi/blob/master/core/src/utilities/helpers/escaping-helpers.js#L54) manually. I only need to revert what that sanitiser does to the string, to get the original input, sanitisation for sanitisations sake is not the intention here.